packages/r/rtmidi/xmake.lua

```diff
@@ -7,6 +7,13 @@ package("rtmidi")
 
     add_versions("6.0.0", "ef7bcda27fee6936b651c29ebe9544c74959d0b1583b716ce80a1c6fea7617f0")
 
+    if is_plat("linux") then
+        add_configs("alsa", {default = false, showmenu = true, description = "Use alsa api on linux.", values={true, false}})
+    end
+    if is_plat("linux", "macosx", "bsd") then
+        add_configs("jack", {default = false, showmenu = true, description = "Use jack api on posix.", values={true, false}})
```

Comment:
remove `showmenu` and `values`, add `type = "boolean"`.

https://github.com/xmake-io/xmake-repo/blob/07f9c9b2c99b0a20b0c023b26967ea734b91c036/packages/f/ffmpeg/xmake.lua#L31